2018-03-07.md

```diff
@@ -0,0 +1,19 @@
+---
+to: All Teams
+subject: New SourceBots Kit Update
+---
+
+Hi!
+
+We're happy to announce a new update to our kits!
+
+# Kit updates
+This release contains many improvements to vision:
+- Completely redo vision to greatly increase its accuracy (now all Cartesian co-ordinates are within 15% of their distance from the camera, all angles are within 5 degrees)
+- Greatly improve marker detection, it is now much less dependent on lighting conditions.
+
+A more detailed change log for this release is available from [the docs](https://docs.sourcebots.co.uk/updates/mar-2018/).
+
+To update your kit, you will need to flash your Pi's SD card. Instructions on [how to flash an SD card](https://docs.sourcebots.co.uk/kit/pi/sd-card/), and the related [download links](https://docs.sourcebots.co.uk/updates/jan-2018/), are available from [the docs](https://docs.sourcebots.co.uk/kit/pi/#updating-your-pi). If you have any issues updating, be sure to consult the documentation, or [contact us](techsupport@sourcebots.co.uk).
```

Comment:
The download page link is incorrect, this will need to be updated with the url from https://github.com/sourcebots/docs/pull/126

Reply:
fixed in e265fc0a9a69b8c234a075058913607b7bf5daa9